spec/models/ems_refresh/parsers/kubernetes_spec.rb

```diff
@@ -49,9 +49,25 @@
                        :image      => {:name => "subname/example", :tag => nil, :image_ref => example_ref},
                        :registry   => {:name => "host.com", :host => "host.com", :port => nil}},
 
+                      {:image_name => "host.com/example",
+                       :image      => {:name => "example", :tag => nil, :image_ref => example_ref},
+                       :registry   => {:name => "host.com", :host => "host.com", :port => nil}},
+
                       {:image_name => "host.com:1234/subname/more/names/example:tag",
                        :image      => {:name => "subname/more/names/example", :tag => "tag", :image_ref => example_ref},
-                       :registry   => {:name => "host.com", :host => "host.com", :port => "1234"}}]
+                       :registry   => {:name => "host.com", :host => "host.com", :port => "1234"}},
+
+                      {:image_name => "localhost:1234/name",
+                       :image      => {:name => "name", :tag => nil, :image_ref => example_ref},
+                       :registry   => {:name => "localhost", :host => "localhost", :port => "1234"}},
+
+                      {:image_name => "localhost:1234/name@sha256:1234567abcdefg",
+                       :image      => {:name => "name", :tag => "sha256:1234567abcdefg", :image_ref => example_ref},
```

Comment:
> Also think of how you're re-building the full_name now... is the digest handled? There's probably an issue there (just because you're not differentiating between tags and digests)

You're right. It needs a fix.